local/src/server/verifier-server.cc

```diff
@@ -484,6 +487,14 @@ ServerReplayFileHandler::proxy_request(YAML::Node const &node)
   if (!errata.is_ok()) {
     return errata;
   }
+  if (!_txn._req.is_http1() && _txn._req._h2_frame_sequence.size() > 0) {
+    std::string frame_sequence;
+    for (size_t i = 0; i < _txn._req._h2_frame_sequence.size(); ++i) {
+      frame_sequence += H2FrameNames[_txn._req._h2_frame_sequence[i]];
+      frame_sequence += " ";
+    }
+    errata.note(S_DIAG, R"(proxy-request node frame sequence: {})", frame_sequence);
+  }
```

Comment:
Same comments here for verifier-server as verifier-client (`empty()`, ranged for, `R"(`).